runtime/gc_glue_java/ArrayletObjectModelBase.hpp

```diff
@@ -53,6 +53,11 @@ class GC_ArrayletObjectModelBase
 	void * _arrayletRangeTop; /**< The top heap range of where discontiguous arraylets are allowed. */
 	MM_MemorySubSpace * _arrayletSubSpace; /**< The only subspace that is allowed to have discontiguous arraylets. */
 	UDATA _largestDesirableArraySpineSize; /**< A cached copy of the subspace's _largestDesirableArraySpineSize to be used when we don't have access to a subspace. */
```

Comment:
While touching this file, and introducing new uintptr_t fields,  let's convert remaining UDATA to uintptr_t (there are a dozen of them). Subclass GC_ArrayletObjectModel is already uniform (uses uintptr_t).

Look for U_32 and similar types, too.